### PR TITLE
Branch delete company

### LINKED ISSUE #63: Add functionality to delete a company

### PR DESCRIPTION
**Pull request by wassilililily**
Closes #63

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCompanyCommand.java
Comment (wassilililily) [inline src/main/java/seedu/address/logic/commands/DeleteCompanyCommand.java:17]: This is just a shortcut way to implement delete company so that the parser still works, will need to have delete contact and delete command inherit from a parent class

**Comment by Quasant** [comment]
I believe your PR is forked off one of the 2 earliest PRs, so it reflects 30 changed files, which makes it hard to do a review.  

**After the earlier PRs are merged**, I recommend this next series of steps. If I made any mistake and there are other alternatives, please sound out! 

Note that I am assuming terminal usage within IDEA and not SourceTree. 

1. Pull from this repo to your local master branch (`git pull remote master`), this command might not work if you did not set your upstream to be this repo
2. Change to your working branch (`git checkout branch-delete-company`)
3. Merge your local master branch to your working branch (`git merge --no-ff master`)
4. Resolve any merge conflicts that arise
5. After the merge is complete, push your working branch to your own repo (`git push origin branch-delete-company`)
6. There is a chance that the files changes might still be 30, as was the case with one of my closed PRs. In that case, close this PR and open a new one, and it should reflect the actual changed files.

**Comment by KengHian** [comment]
> I believe your PR is forked off one of the 2 earliest PRs, so it reflects 30 changed files, which makes it hard to do a review.
> 
> **After the earlier PRs are merged**, I recommend this next series of steps. If I made any mistake and there are other alternatives, please sound out!
> 
> Note that I am assuming terminal usage within IDEA and not SourceTree.
> 
> 1. Pull from this repo to your local master branch (`git pull remote master`), this command might not work if you did not set your upstream to be this repo
> 2. Change to your working branch (`git checkout branch-delete-company`)
> 3. Merge your local master branch to your working branch (`git merge --no-ff master`)
> 4. Resolve any merge conflicts that arise
> 5. After the merge is complete, push your working branch to your own repo (`git push origin branch-delete-company`)
> 6. There is a chance that the files changes might still be 30, as was the case with one of my closed PRs. In that case, close this PR and open a new one, and it should reflect the actual changed files.

Actually, this is because Ze Wei had to merge Jun Boon and Alek's changes to his branch as he had to build on top of their code. 

So we have to wait for Jun Boon and Alek's PR to be merged to master, before (hopefully) this PR only capture Ze Wei's changes.

File: src/main/java/seedu/address/logic/commands/DeleteCompanyCommand.java
Comment (KengHian) [reply src/main/java/seedu/address/logic/commands/DeleteCompanyCommand.java:17]: Alright,  I think to keep track of this, you should add a // TODO comments in this file, and add this as an issue. 

After which this conversation can be resolved.

**Review by KengHian — APPROVED** [review]
Not much to add, LGTM

File: src/test/java/seedu/address/logic/parser/DeleteContactCommandParserTest.java
Comment (Quasant) [inline src/test/java/seedu/address/logic/parser/DeleteContactCommandParserTest.java:1]: I believe this was originally the test for the DeleteCommandParser. Just make a note that this file will probably have to change back to the original name, when Alek's PR is merged.

**Review by Quasant — COMMENTED** [review]
Yea seems fine to me, just note that modifications might be needed later on after the abstract `DeleteCommand` has been implemented.

**Review by Quasant — APPROVED** [review]
Forget to change from comment mode to approve mode.

**Pull request merged**
